my.css

```diff
@@ -1,6 +1,10 @@
+html {
+  font-size: 16px; 
+}
+
+// magic number
 body {
-  background: #000000;
-  font-size:  12px;
+  background: #123456;
```

Comment:
This is a magic number. Where does this come from?
A CSS variable can help here if supported by the project. Otherwise a comment or a sass variable if such a code is used in an environment supporting them.

Reply:
that's new and nice